baseclass/src/test/java/com/google/apigee/calloutsbaseclass/tests/CalloutsBaseTest.java

```diff
@@ -0,0 +1,215 @@
+package com.google.apigee.calloutsbaseclass.tests;
```

Comment:
tests is not necessary.  Usually tests are in the same packages, but different directories.